

/usr/src/25/kernel/crash.c:90: undefined reference to `kmap_atomic_pfn'
/usr/src/25/kernel/crash.c:92: undefined reference to `kunmap_atomic'

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/crash.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletion(-)

diff -puN kernel/crash.c~crashdump-kmap-build-fix kernel/crash.c
--- 25/kernel/crash.c~crashdump-kmap-build-fix	Tue Sep 21 17:08:38 2004
+++ 25-akpm/kernel/crash.c	Tue Sep 21 17:15:34 2004
@@ -9,9 +9,11 @@
 #include <linux/kexec.h>
 #include <linux/errno.h>
 #include <linux/proc_fs.h>
+#include <linux/highmem.h>
+#include <linux/crash_dump.h>
+
 #include <asm/io.h>
 #include <asm/uaccess.h>
-#include <asm/highmem.h>
 
 /*
  * Enable kexec reboot upon panic; for dumping
_
